src/config/hoard.rs

```diff
@@ -122,7 +133,10 @@ impl Pile {
                 error: err,
             })?;
         } else {
-            log::warn!("{} is not a file or directory", src.to_string_lossy());
+            tracing::warn!(
+                source = src.to_string_lossy().as_ref(),
+                "source is not a file or directory",
+            );
```

Comment:
Should an actual error be returned here?
(Perhaps a question for a later time)

Reply:
I think the warning is appropriate. Copying things like symbolic links seems too likely to cause problems, but I don't want it to break things.